index.html

```diff
@@ -11,7 +11,7 @@
   <body>
     <div id="app"></div>
     <script type="module">
-      import ChooseArea from "./src/ChooseArea.svelte";
+      import ChooseArea from "./src/pages/ChooseArea.svelte";
```

Comment:
Oh yeah, I forgot to mention in the PR description.. I also renamed/organized a bunch of Svelte components as part of this PR. Apologies; it makes the diff larger. I can split into a separate PR if useful. I just wound up getting annoyed at the too-flat structure in the middle of working on this PR.